src/renderer/react/app/nodes/NodeManager.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { nodeShape } from 'react/shapes/node';
+import * as nodeActions from 'redux/stores/nodes/actions';
+import isAuthenticated from 'react/hoc/isAuthenticated';
+
+import NodeManagerList from './nodeList/NodeManagerList';
+import PromptNewNodeModal from './editNodeModal/EditNodeModal';
+
+@isAuthenticated
+@connect(state => ({
+  nodeList: state.nodes,
```

Comment:
I don't know what y'all think but I think we should try using mapDispatchToProps too !